public/static/locales/en/donate.json

```diff
@@ -13,7 +13,13 @@
   "taxDeductionNotAvailableForProject": "Tax deduction is not yet available for this project",
   "youWillReceiveTaxDeduction": "You will receive a tax deduction receipt for ",
   "toProjectByOrganization": "To {{projectName}} by {{organizationName}}",
-  "fortreeCountTrees": "for {{treeCount}} Trees",
+  "fortreeCountTreesnone": "for {{treeCount}} Trees",
+  "fortreeCountTreesdaily": "for {{treeCount}} Trees every day",
+  "fortreeCountTreesweekly": "for {{treeCount}} Trees every week",
+  "fortreeCountTreesmonthly": "for {{treeCount}} Trees every month",
+  "fortreeCountTreesquaterly": "for {{treeCount}} Trees every three months",
+  "fortreeCountTreessemiyearly": "for {{treeCount}} Trees every six months",
+  "fortreeCountTreesyearly": "for {{treeCount}} Trees every year",
```

Comment:
@rishsingh07 please make this one single item.

`{cost} {currency} for {treecount} Trees {frequency}`

Eg: 50 EUR for 50 Trees every month

if frequency = Once, 
`50 EUR for 50 Trees`

We only need Once, every month, and  every year.

Please remove other options for now.

Reply:
Appending `{frequency}` to this phrase works in German language - hopefully also in other languages - we probably have to explain which resources can be inserted for `{frequency}` when pushing this to LingoHub.

Reply:
We just have one variable value which we are using to show the clickable options, we have to use the same variable to show to text as well, hence we cannot do it this way.